sdk/nodejs/resource.ts

```diff
@@ -756,6 +756,11 @@ export interface ResourceOptions {
      */
     aliases?: Input<URN | Alias>[];
 
+    /**
+     * A list of output properties to mark as secret.
+     */
+    additionalSecretOutputs?: string[];
```

Comment:
Hm; I'm not sure if these belong on the public interface, and if they do, whether they belong on both custom and component options, or just one -- @Frassle do you know?

Reply:
My plan was to follow this https://github.com/pulumi/pulumi/blob/master/sdk/go/pulumi/resource.go#L439
I might have conflated two interfaces, though

Reply:
Yeh I think this should be here. Just check mergeOptions is ok with this.